bot/database/roles.py

```diff
@@ -0,0 +1,72 @@
+from textwrap import dedent
+
+from discord import Guild, Role
+from loguru import logger
+
+from bot.core.bot import Bot
+from bot.database import DBTable, Database
+
+
+class Roles(DBTable):
+    """
+    This table stores all guild-specific roles:
+    * `default` role (column is named `_default` to avoid SQL confusion)
+    * `muted` role
+    * `staff` role
+    Under the single `serverid` column
+    """
+    populate_command = dedent("""
+        CREATE TABLE IF NOT EXISTS roles (
+            serverid NUMERIC(40) UNIQUE NOT NULL,
+            _default NUMERIC(40) DEFAULT 0,
```

Comment:
What's this default role?

Reply:
it's the equivalent of verified role, the role which every user has, in our server, it's `@Member` role. I named it `_default` rather than just `default` because `default` is SQL command